ledger/heavyserver/heavysync.go

```diff
@@ -48,6 +51,26 @@ type syncstate struct {
 	// insyncend core.PulseNumber
 	syncpulse *core.PulseNumber
 	insync    bool
+	timer     *time.Timer
+}
+
+func (s *syncstate) resetTimeout(ctx context.Context, pn core.PulseNumber) {
```

Comment:
please rename to other name, and allow to pass timeout value (value should be received from configuration)

Reply:
also we don't need PulseNumber here